src/pages/SinglePlaylistPage/SinglePlaylist.jsx

```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { Navbar,Sidebar, EmptyPage, VideoCard } from "../../Components/index";
+import { getPlaylistFromPlaylists } from "../../ApiCalls";
+
+export const SinglePlaylist = () => {
+    const [playlistVideos, setPlaylistVideos] = useState();
+    const [loader, setLoader] = useState(true);
+    const {playlistId} = useParams();
+
+    useEffect(() => {
+        getPlaylistFromPlaylists (playlistId, setPlaylistVideos, setLoader);
+    }, [playlistVideos]);
+
+    if(loader) {
+        return <h1>Loading ....</h1>
```

Comment:
For UX I think it would be better to show loader for only components that are loading and keep Navbar and Sidebar